backend/src/chains/similarity_retriever_chain.py

```diff
@@ -93,3 +95,54 @@ def create_llm_chain(self) -> None:
         self.llm_chain = llm_chain_with_source
 
         return
+
+
+if __name__ == "__main__":
+    load_dotenv()
+
+    # from langchain_google_vertexai import ChatVertexAI
+    # llm = ChatVertexAI(model_name="gemini-1.5-pro")
+
+    llm = ChatGoogleGenerativeAI(model="gemini-pro", temperature=1)
+
+    prompt_template_str = """
```

Comment:
Can you shift this to some central folder? Maybe use backend/src/prompts to collect all these prompts rather than listing them one by one

Reply:
This make it easier to update and test different prompts

Reply:
Done, the prompt templates are in a separate folder now